plugins/org.jkiss.dbeaver.ext.ui.tipoftheday/tips.xml

```diff
@@ -158,6 +165,16 @@
 	<tip product="ce">With DBeaver PRO, you can compare schemas, including all metadata objects, in one or different databases.
 		 Learn more about the powerful schema comparison in DBeaver PRO
 		 <a href="https://dbeaver.com/docs/wiki/Schema-compare/?utm_source=ce&amp;utm_medium=cpc&amp;utm_campaign=tips">here</a>.</tip>
+	<tip product="ce">To switch ER Diagram notation, right-click on a diagram, go to the <b>Notation</b> item, and select the one you need — for example, Crows Foot.</tip>
```

Comment:
Use a suggestion provided below. It's also better to keep these two tips together, so they won't get lost